test/gui/tst_sharing/test.feature

```diff
@@ -53,3 +53,36 @@ Feature: Sharing
         And the user toggles the password protection using the client-UI
         And the user toggles the password protection using the client-UI
         Then the progress indicator should not be visible in the client-UI
+
+    Scenario: unshare a reshared file
+        Given the setting "shareapi_auto_accept_share" on the server of app "core" has been set to "no"
+    	And the administrator on the server has set the default folder for received shares to "Shares"
```

Comment:
fix indentation here

Reply:
This is weird.  Locally the indention is fine but git is showing the wrong indentation.

Reply:
check tabs vs spaces